packages/rrweb/src/record/observer.ts

```diff
@@ -971,17 +974,26 @@ export function initObservers(
     o.blockClass,
     o.mirror,
   );
+
+  const curr_window = o.doc.defaultView,  // basically document.window
+
   const styleSheetObserver = initStyleSheetObserver(
     o.styleSheetRuleCb,
+    curr_window,
     o.mirror,
   );
   const styleDeclarationObserver = initStyleDeclarationObserver(
     o.styleDeclarationCb,
+    curr_window,
```

Comment:
JavaScript conventions
```suggestion
  const currentWindow = o.doc.defaultView,  // basically document.window

  const styleSheetObserver = initStyleSheetObserver(
    o.styleSheetRuleCb,
    currentWindow,
    o.mirror,
  );
  const styleDeclarationObserver = initStyleDeclarationObserver(
    o.styleDeclarationCb,
    currentWindow,
```

Reply:
Thanks! My bad.